.meteor/packages

```diff
@@ -28,7 +31,7 @@ react-template-helper
 react-meteor-data
 
 # data
-mongo@1.4.2                  # The database Meteor supports right now
+mongo@1.4.5                   # The database Meteor supports right now
```

Comment:
Are all this package version changes necessary for this PR?

Reply:
Nope, I just got lazy :see_no_evil:. Fixing this now… Thanks ;)